.github/CONTRIBUTING.md

```diff
@@ -133,7 +136,18 @@ logically independent patches which build on each other.  This makes large
 changes easier to review and approve which speeds up the merging process.
 * Try to keep pull requests simple. Simple code with comments is much easier
 to review and approve.
+* All proposed changes must be approved by a ZFS on Linux organization member.
+
+#### Tests and Benchmarks
+* Every pull request will by tested by the buildbot on multiple platforms by running the [zfs-tests.sh](
+https://github.com/zfsonlinux/zfs/wiki/Building-ZFS#running-zloopsh-and-zfs-testssh) and [zloop.sh](
+https://github.com/zfsonlinux/zfs/wiki/Building-ZFS#running-zloopsh-and-zfs-testssh) test suites.
+* To verify your changes conform to the [style guidelines](
+https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#style-guides
+) please run `make checkstyle`, resolve any warnings, and then open the pull request.
```

Comment:
nit: add a comma after "style guidelines", outside of the link.

I'm not sure that the "and then open the pull request" bit is required, or even the bit before it:
```
To verify your changes conform to the [style guidelines](
https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#style-guides
), please run `make checkstyle`
````

Reply:
Agreed!